benchmark.js

```diff
@@ -179,36 +215,12 @@ var Benchmark = (function() {
       saveStorage();
       this.runNextRound();
     },
-    sampleMemory: function() {
-      if (NO_SECURITY) {
-        forceCollectors();
-        var memoryReporter = Components.classes["@mozilla.org/memory-reporter-manager;1"].getService(Components.interfaces.nsIMemoryReporterManager);
-
-        var jsObjectsSize = {};
-        var jsStringsSize = {};
-        var jsOtherSize = {};
-        var domSize = {};
-        var styleSize = {};
-        var otherSize = {};
-        var totalSize = {};
-        var jsMilliseconds = {};
-        var nonJSMilliseconds = {};
-
-        try {
-          memoryReporter.sizeOfTab(window.parent.window, jsObjectsSize, jsStringsSize, jsOtherSize,
-            domSize, styleSize, otherSize, totalSize, jsMilliseconds, nonJSMilliseconds);
-        } catch (e) {
-          console.log(e);
-        }
-        storage.current.totalSize.push(totalSize.value);
-        storage.current.domSize.push(domSize.value);
-        storage.current.styleSize.push(styleSize.value);
-        storage.current.jsObjectsSize.push(jsObjectsSize.value);
-        storage.current.jsStringsSize.push(jsStringsSize.value);
-        storage.current.jsOtherSize.push(jsOtherSize.value);
-        storage.current.otherSize.push(otherSize.value);
-        saveStorage();
+    sampleMemoryToStorage: function() {
+      var mem = sampleMemory();
+      for (var p in mem) {
+        storage.current[p] = mem[p];
```

Comment:
This value should be pushed to the array no replace it. I have fixed this in my upcoming changes.